src/gaen/nativeModule.ts

```diff
@@ -101,6 +101,10 @@ export const fetchLastExposureDetectionDate = async (): Promise<Posix | null> =>
   }
 }
 
+export const checkForNewExposures = async (): Promise<void> => {
+  return await exposureHistoryModule.detectExposures()
```

Comment:
The method `DebugMenuModule#detectExposuresNow` is no longer needed, right?
i can remove it from the UI if so